salt/metalk8s/registry/installed.sls

```diff
@@ -8,7 +8,7 @@
 Inject OCI registry image:
   containerd.image_managed:
     - name: docker.io/library/registry:2.7.1
-    - archive_path: {{ metalk8s.iso_root_path }}/images/registry-2.7.1.tar
+    - archive_path: {{ metalk8s.iso_root_path[saltenv] }}/images/registry-2.7.1.tar
```

Comment:
Could we use a path relative to the SLS path, which is (if I remember correctly) available in the Jinja rendering context?

Reply:
No we can't because the path we got with `slspath` it's the relative path from `file_roots` we can not get the absolute path. (here \slspath` will be `metalk8s/registry` I think)

OR 

Btw we can do something like `salt.config.get('file_roots')[saltenv]` but then we get a list and we can maybe assume that it's only a list of one single element, but I don't really like this method